src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,239 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+
+/**
+* Represents the static data of a user.
+*/
+public class User {
+    private final String userID;
+    private final String username;
+    private final String firstName;
+    private final String lastName;
+    private final String profilePicture;
```

Comment:
As for "String profilePicture", should we use "profilePictureUrl"? If it is a profile picture, then I expect the type should be a picture type.

Reply:
Yeah, I was thinking about this while coding it. I think I'll make this change, thanks!